src/components/Dashboard/Order/__tests__/OrdersPage.test.js

```diff
@@ -0,0 +1,49 @@
+import { OrdersPage } from '../OrdersPage';
+
+const defaultProps = {
+  orders: [],
+  loaded: false,
+  getOrdersFromServer: jest.fn(),
+  token: '1234',
+  userId: '1a2a',
+};
+const setup = buildSetup(OrdersPage, defaultProps);
+
+describe('<OrdersPage />', () => {
+  it('should render without crash', () => {
+    const { wrapper } = setup();
+    expect(wrapper).toMatchSnapshot();
+  });
+  it('should call getOrdersFromServer when component did mount', () => {
+    const {
+      wrapper,
+      props: { token, userId },
+    } = setup();
+    expect(wrapper.instance().props.getOrdersFromServer).toHaveBeenCalledWith(userId, token);
+  });
+  it('state should be', () => {
```

Comment:
state should be what?

Also, testing that state is the same as hardcoded values gives us little value, as it will break only when we manually change the initial state. But in such case it was our intention to do so.